src/components/announcements/IntroProjectTags.tsx

```diff
@@ -1,10 +1,21 @@
+import { Trans } from '@lingui/macro'
+import ProjectTagsRow from 'components/ProjectTagsRow'
+import { projectTagOptions } from 'models/project-tags'
+
 export default function IntroProjectTags() {
   return (
     <div>
-      <h1>Introducing project tags</h1>
-      Project tags are fun and great and fun
+      <h1>
+        <Trans>
+          <span className="text-juice-500">NEW:</span> Project tags
+        </Trans>
+      </h1>
+      <ProjectTagsRow tags={[...projectTagOptions]} />
       <br />
```

Comment:
**nitpick** might we avoid the `br` 😅

Reply:
ahhh for sure lol. sorry I'm always dropping <br/>s